diff --git a/Makefile b/Makefile
index f974fa7..5f370fe 100644
--- a/Makefile
+++ b/Makefile
@@ -43,15 +43,18 @@ LIBNAME=$(LIB).so.1.0

 CXX=g++
 CC=gcc
+CP=cp
 CFLAGS=$(CCFLAGS)

 # make all
 # reinstall the library after each recompilation
-all: ArduiPi_OLED install
+all: ArduiPi_OLED

 # Make the library
 ArduiPi_OLED: ArduiPi_OLED.o Adafruit_GFX.o bcm2835.o Wrapper.o
        $(CXX) -shared -Wl,-soname,$(LIB).so.1 $(CFLAGS) $(LDFLAGS)  -o ${LIBNAME} $^ -li2c
+       $(CP) examples/oled_demo.cpp .
+       $(CXX) $(CFLAGS) -Wall oled_demo.cpp -o oled_demo ${LIBNAME}

 # Library parts (use -fno-rtti flag to avoid link problem)
 ArduiPi_OLED.o: ArduiPi_OLED.cpp